sdks/go/pkg/beam/core/runtime/exec/coder.go

```diff
@@ -134,9 +134,15 @@ type bytesEncoder struct{}
 
 func (*bytesEncoder) Encode(val FullValue, w io.Writer) error {
 	// Encoding: size (varint) + raw data
-
-	data := reflect.ValueOf(val.Elm).Convert(reflectx.ByteSlice).Interface().([]byte)
-	size := len(data)
```

Comment:
It seems this line got lost in editing. I don't think the code compiles. The Go presubmit test essentially runs "go test ./..." and should be green.